Dockerfile

```diff
@@ -10,18 +10,26 @@ RUN set -ex; \
   curl -sL https://deb.nodesource.com/setup_20.x | bash -; \
   curl https://raw.githubusercontent.com/kadwanev/retry/0b65e6b7f54ed36b492910470157e180bbcc3c84/retry -o /usr/bin/retry; \
   chmod +x /usr/bin/retry; \
+  # Add Ondrej PHP PPA for PHP 8.2
+  add-apt-repository ppa:ondrej/php -y; \
   apt-get update; \
   apt-get install --no-install-recommends --no-install-suggests -y \
-  php php-apcu php-bcmath php-dom php-ctype php-curl php-exif php-fileinfo php-fpm \
-  php-gd php-gmp php-iconv php-intl php-json php-mbstring php-mysqlnd php-soap \
-  php-redis php-mysqli php-opcache php-pdo php-phar php-posix php-simplexml \
-  php-sockets php-sqlite3 php-tidy php-tokenizer php-xml php-xmlwriter php-zip \
+  php8.2 php8.2-apcu php8.2-bcmath php8.2-dom php8.2-ctype php8.2-curl php8.2-exif php8.2-fileinfo php8.2-fpm \
+  php8.2-gd php8.2-gmp php8.2-iconv php8.2-intl php-json php8.2-mbstring php8.2-mysqlnd php8.2-mysql php8.2-soap \
+  php8.2-redis php8.2-mysqli php8.2-opcache php8.2-pdo php8.2-phar php8.2-posix php8.2-pdo-sqlite php8.2-simplexml \
+  php8.2-sockets php8.2-sqlite3 php8.2-tidy php8.2-tokenizer php8.2-xml php8.2-xmlwriter php8.2-zip php8.2-dev \
   php-pear libgd-tools \
   nodejs \
   git unzip; \
+  npm install -g pnpm wrangler@3.56.0 firebase-tools; \
```

Comment:
Hi @francois-codes  Here we are installing the necessary dependencies for the FE. we were running this in the CI before. Please Update me If I am missing any dependencies. and I verified the CI and it is working fine.

Reply:
yes confirmed. it's working fine.